python/pipeline/experiment.py

```diff
@@ -593,4 +593,66 @@ class AutoProcessing(dj.Manual):
     autosegment=false   :boolean       # segment somas in the first channel with default method
     """
 
+@schema
+class ProjectorColor(dj.Lookup):
```

Comment:
this does not agree with the ProjectorColor.m file

Reply:
I think you need to modify it here and probably change the contents of the table

Reply:
Updated